documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -99,9 +99,9 @@ This does not have to be the name of the Kafka cluster.
 It can be the label assigned to the `KafkaTopic` resource.
 If you deploy more than one Topic Operator, the labels must be unique for each.
 That is, the operators cannot manage the same resources.
-<4> The host and port pair of the address to connect to the ZooKeeper cluster.
+<4> (ZooKeeper) The host and port pair of the address to connect to the ZooKeeper cluster.
 This must be the same ZooKeeper cluster that your Kafka cluster is using.
-<5> The ZooKeeper session timeout, in milliseconds.
+<5> (ZooKeeper) The ZooKeeper session timeout, in milliseconds.
 The default is `18000` (18 seconds).
```

Comment:
Should we mention here the options ignored in Unidirectional TO as well?

Reply:
Instead of this, I've added an example config for the UTO.